client/me/concierge/main.js

```diff
@@ -1,13 +1,42 @@
+/** @format */
+
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 
-const ConciergeMain = () => {
-	return (
-		<div>{ 'Concierge chats page placeholder.' }</div>
-	);
-};
+/**
+ * Internal dependencies
+ */
+import { fetchConciergeShifts } from 'state/concierge/actions';
+import { getConciergeShifts, isFetchingConciergeShifts } from 'state/selectors';
+
+class ConciergeMain extends Component {
+	componentDidMount() {
+		// TODO: pass in the real WP.com concierge schedule id.
+		this.props.fetchConciergeShifts( 123 );
```

Comment:
Any reason we're not using a [QueryComponent](https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#query-components) instead?

Reply:
I was having a false impression that it wouldn't be necessary to use a query component now that we are leveraging the data layer. Obviously I was wrong and it is still in our best practice, so I've added it in f1bfad6 .

Reply:
If there's a special need we definitely can use the actions directly, but it felt like a good fit here.